src/test/java/com/snack/news/service/AdminServiceTest.java

```diff
@@ -91,17 +95,20 @@
 		when(newsRepository.findAll(any(Pageable.class))).thenReturn(new PageImpl<>(newsList));
 
 		adminService.getNewsList(1);
+		verify(newsRepository).findAll(any(Pageable.class));
 	}
 
 	@Test
-	public void 뉴스를_삭제할_수_있다() {
+	@DisplayName("뉴스를 삭제할 수 있다")
+	void deleteNewsTest() {
 		adminService.deleteNews(anyLong());
 		verify(newsRepository).deleteById(anyLong());
 	}
 
-	@Test(expected = NewsNotFoundException.class)
-	public void 뉴스를_삭제_요청시_뉴스ID가_존재히지_않는다면_예외가_발생한다() {
+	@Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4863700)